web-client/iron-remote-gui/package.json

```diff
@@ -16,16 +16,29 @@
     "build-alone": "vite build",
     "pre-build": "node ./pre-build.js",
     "preview": "vite preview",
-    "check": "svelte-check --tsconfig ./tsconfig.json"
+    "check": "svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+    "lint": "npm run lint:prettier && npm run lint:eslint",
+    "lint:prettier": "prettier . --check",
+    "lint:eslint": "eslint src/**",
+    "format": "prettier . --write ."
```

Comment:
Here are the scripts:
- `npm run check` will verify svelte and typescript
- `npm run lint` will very eslint and prettier
- `npm run format` will actually run the formatter